src/components/AppNavigation/Settings.vue

```diff
@@ -20,10 +20,20 @@
   -->
 
 <template>
-	<AppNavigationSettings :title="settingsTitle">
-		<ul class="settings-fieldset-interior">
+<!--  <AppNavigationSettings icon="icon-settings"
```

Comment:
calendar doesnt have a navigation or sidebar component, so need to find a way where to add this